src/components/vulnerabilities/Form.js

```diff
@@ -89,6 +95,16 @@ const VulnerabilityForm = ({
         return false;
     }
 
+    const isOwaspProject = (elements, id) => {
+        let metrics = ProjectVulnerabilityMetrics[0].id;
+        for (const el of elements) {
+            if (el.id === parseInt(id)) {
+                metrics = el.vulnerability_metrics;
+            }
+        }
+        return (ProjectVulnerabilityMetrics[1].id === metrics);
+    }
```

Comment:
I am sure there is a simpler way to answer the question if a project is using owasp rr or not. I might refactor after merge.

Reply:
Thank you!